libr/core/cconfig.c

```diff
@@ -558,6 +558,21 @@ static void update_asmarch_options(RCore *core, RConfigNode *node) {
 	}
 }
 
+static void update_asmbits_options(RCore *core, RConfigNode *node) {
+	if (core && core->rasm && core->rasm->cur && node) {
+		char buf[20];
+		int bits = core->rasm->cur->bits;
+		int i;
+		r_list_purge (node->options);
+		for (i = 1; i <= bits; i <<= 1) {
+			if (i & bits) {
+				snprintf (buf, 20, "%d", i);
```

Comment:
remove this line, but if you need to pass fixed size buffers, is better to use sizeof than duplicuant constant values

Reply:
Understood, I have replaced snprintf with r_str_newf.